### PR TITLE
Miscaellenous fixes

### PR DESCRIPTION
**Pull request by jeromer**
A couple of fixes for annoying issues with amqp and SSL connections with Request

:)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -25,8 +25,11 @@ mock==1.0.1
 psycogreen==1.0
 psycopg2==2.5.1
 pytz==2014.10
+pyOpenSSL
+ndg-httpsclient
+pyasn1
```
Comment (JeanFred) [inline requirements.txt]: Could you please pin those to a known working version @jeromer ? :)

**Comment by JeanFred** [comment]
Thanks for this! Just one comment.

**Comment by jeromer** [comment]
Here you go :)

File: requirements.txt [hunk 2]
```diff
@@ -2,7 +2,7 @@ Django==1.6.11
 Markdown==2.5
 PyJWT==0.1.2
 South==1.0
-amqp>=1.4.9
+amqp<2.0
```
Comment (JeanFred) [inline requirements.txt]: I was wondering whether it would make sense to have `>=1.4.0,<2.0` but there are no other releases in between. Good :)

**Pull request merged**